prow/cmd/deck/static/script.js

```diff
@@ -26,16 +26,16 @@ function optionsForRepo(repo) {
         states: {},
     };
```

Comment:
Why not do an initialization pass to set `build.author` and `build.number`, and avoid the construct of `build.spec.refs.pulls[0]` elsewhere?

Reply:
This should still be done-- getting the hnumber out should be simpler.